index.html

```diff
@@ -73,7 +73,7 @@ <h1 class="title">2048</h1>
     </div>
 
     <p class="game-explanation">
-      <strong class="important">How to play:</strong> Use your <strong>arrow keys</strong> to move the tiles. When two tiles with the same number touch, they <strong>merge into one!</strong>
+      <strong class="important">ਕਿਵੇਂ ਖੇਡਣਾ ਹੈ: </ strong> ਟਾਈਲਾਂ ਨੂੰ ਮੂਵ ਕਰਨ ਲਈ ਆਪਣੇ <strong> ਐਰੋ ਬਟਨ </ strong> ਦੀ ਵਰਤੋਂ ਕਰੋ. ਜਦੋਂ ਇਕੋ ਨੰਬਰ ਦੇ ਨਾਲ ਦੋ ਟਾਈਲਾਂ ਛੋਹ ਜਾਂਦੀਆਂ ਹਨ, ਤਾਂ ਉਹ <strong> ਇੱਕ ਵਿੱਚ ਲੀਨ ਹੋ ਜਾਂਦੀਆਂ ਹਨ </ strong>
```

Comment:
The closing `</strong>` tags shouldn't have spaces

Reply:
updated